script.js

```diff
@@ -0,0 +1,69 @@
+const hamburger = document.querySelector('.hamburger');
+const navi = document.querySelector('.navi');
+const disappear = document.querySelector('.disappear');
+const headerDisappear = document.querySelector('.header-text');
+const headerDesktop = document.querySelector('.header-desktop');
+const projectID = document.querySelector('#project-section');
+const aboutID = document.querySelector('#about-section');
+const contactID = document.querySelector('#contact-section');
+const lineSelector = document.querySelector('.line');
+const lineSelector1 = document.querySelector('.line1');
+const lineSelector2 = document.querySelector('.line2');
+const cancel1 = document.querySelector('.cancel1');
+const cancel2 = document.querySelector('.cancel2');
+const sticky = headerDesktop.offsetTop;
+
+function addLine() {
+  lineSelector.classList.add('underline');
+}
+function addLine1() {
+  lineSelector1.classList.add('underline');
+}
+function addLine2() {
+  lineSelector2.classList.add('underline');
+}
+
+function removeLine() {
+  lineSelector.classList.remove('underline');
+}
+function removeLine1() {
+  lineSelector1.classList.remove('underline');
+}
+function removeLine2() {
+  lineSelector2.classList.remove('underline');
+}
+
+function scrollFunction() {
+  if (window.pageYOffset > sticky) {
+    headerDesktop.classList.add('scroll');
+    headerDesktop.classList.add('sticky');
+  } else {
+    headerDesktop.classList.remove('scroll');
+    headerDesktop.classList.add('sticky');
+  }
+}
+
+function mobileMenu() {
+  headerDisappear.classList.toggle('active');
+  disappear.classList.toggle('active');
+  hamburger.classList.toggle('active');
+  navi.classList.toggle('active');
+}
+
+hamburger.addEventListener('click', mobileMenu);
+window.onscroll = () => { scrollFunction(); };
+cancel1.addEventListener('click', mobileMenu);
+cancel2.addEventListener('click', mobileMenu);
+
+hamburger.addEventListener('click', mobileMenu);
+window.onscroll = () => { scrollFunction(); };
+cancel1.addEventListener('click', mobileMenu);
+cancel2.addEventListener('click', mobileMenu);
+
+projectID.addEventListener('mouseover', addLine);
+aboutID.addEventListener('mouseover', addLine1);
+contactID.addEventListener('mouseover', addLine2);
+
+projectID.addEventListener('mouseout', removeLine);
+aboutID.addEventListener('mouseout', removeLine1);
+contactID.addEventListener('mouseout', removeLine2);
```

Comment:
- [ ] Please know that when I click on `Portfolio` in the mobile menu it doesn't disappear and the X mark disappears, so I have no way of closing the menu.
![image](https://user-images.githubusercontent.com/20567503/195377785-972a778e-9ad9-441e-9c3b-44a486b8b843.png)

- [ ] Also as you can see in the above screenshot, the menu isn't covering the whole screen you can see under it, please make sure it's covering the whole screen.
- [ ] I suggest that you place the X button inside the mobile menu and not in the header that way it won't disappear even if it's scrolled.